components/style/properties/gecko.mako.rs

```diff
@@ -3345,6 +3346,23 @@ fn static_assert() {
         self.copy_transition_property_from(other)
     }
 
+    pub fn set_transform_style(&mut self, v: TransformStyle) {
```

Comment:
Maybe note in a comment that this is done because of the `preserve3d` / `preserve-3d` stuff, which is why we can use one of our crappy macro helpers? Just so I don't waste time trying to rewrite it in the future :)

Reply:
err, _can't_ of course